app/src/main/java/org/oppia/app/story/StoryFragmentPresenter.kt

```diff
@@ -52,6 +65,8 @@ class StoryFragmentPresenter @Inject constructor(
   }
 
   private fun createRecyclerViewAdapter(): BindableAdapter<StoryItemViewModel> {
+    smoothScrollToPosition(2)
```

Comment:
This is hardcoded please update it to be dynamic.

Reply:
This was written for dummy testing and has been removed as it was not needed anymore.